caiman/summary_images.py

```diff
@@ -694,20 +694,30 @@ def local_correlations_movie(file_name,
 
 def local_correlations_movie_offline(file_name,
                                      Tot_frames=None,
-                                     fr: int = 10,
+                                     fr: float = 10.,
                                      window: int = 30,
                                      stride: int = 3,
-                                     swap_dim: bool = True,
+                                     swap_dim: bool = False,
                                      eight_neighbours: bool = True,
                                      order_mean: int = 1,
                                      ismulticolor: bool = False,
-                                     dview=None):
+                                     dview=None,
+                                     remove_baseline: bool = False,
+                                     winSize_baseline: int = 5,
+                                     quantil_min_baseline: float = 8):
 
     if Tot_frames is None:
-        Tot_frames = cm.load(file_name).shape[0]
+        _, Tot_frames = get_file_size(file_name)
```

Comment:
We have a specific method for this that does not load the entire file in memory.
Problems will arise for non standard hdf5/NWB files